.github/workflows/website.yml

```
@@ -5,70 +5,133 @@ on:
+      STORYBLOK_PREVIEW_TOKEN: ${{ secrets.TF_STAGING_STORYBLOK_PREVIEW_TOKEN }}
+      HOME: /root
+
+    steps:
+      - uses: actions/checkout@v5
         with:
-          terraform_version: 1.6.6
+          ref: ${{ github.head_ref }}
+          persist-credentials: false
+
+      - name: Initialize repository
+        uses: ./.github/workflows/actions/init
```

actionlint (1.7.10)

could not parse action metadata in "/home/jailuser/git/.github/workflows/actions/init": line 13: cannot unmarshal !!str `false` into bool

(action)